_specification-v1/sass.md

```diff
@@ -93,11 +88,11 @@ As Sass has limited encapsulation, any Sass (e.g. mixin, function, variable) tha
 
 ## Sass Includes
 
-If a component contains SCSS files other than the main file listed in `bower.json`:
+If a component contains SCSS files other than the root `main.scss` file:
 
 - Files for import **must** be prefixed with an underscore, to [indicate privacy](#private-sass).
 - All import statements **must** be imported before any other Sass.
-- All import statements **should** be in the components’s main file.
+- All import statements **should** be in the components’s `main.scss` file.
 - Sass variables, mixins, and functions **should** be in their own files.
```

Comment:
The index scss file is now a **must**?

Reply:
Do you mean we are currently missing a requirement in the spec?
Like this?
```suggestion
- Sass variables, mixins, and functions **should** be in their own files.
- `_index.scss` **must** exist and only import the `main.scss` file.
```